.github/workflows/json_verify.yml

```diff
@@ -1,30 +0,0 @@
-name: CI
```

Comment:
why?

Reply:
It's still there; just moved it to be combined into the other workflow